lernfragen.md

```diff
@@ -160,3 +176,19 @@ HCI Lernfragen
 ##### 79. Ein gutes Hilfesystem ist eine unerlässliche Bedingung für selbstbeschreibungfähige Systeme.
 
 ##### 80. Körperbehinderungen verursachen erhöhte Transformationsdistanzen in den unteren Ebenen des 6- Ebenen-Modells.
+
+#####______________________________________________________
+##### 81. Hintergrundbilder sollten grundsätzlich ausblendbar sein.
+Richtig.
+
+##### 82. Je genauer ein GUI-Prototyp ausgeführt ist, desto besser kann man ihn in der Evaluation einsetzen
```

Comment:
Hier bin Ich verwirrt... in HCI-06-GUI-Prototypen Seite 20 spricht sie von "High-Fidelity-Prototypen ermöglichen präzisere Bewertung" ... davor sagt Sie "Je einfacher der Prototyp umso besser fokusiert er den Blick auf die essentielle Interaktion" Ich schätze hier wäre nein besser, denn: http://www.usabilityfirst.com/glossary/high-fidelity-prototype/